analyzer/src/main/resources/META-INF/services/com.here.ort.analyzer.PackageManagerFactory

```diff
@@ -1,11 +1,11 @@
 com.here.ort.analyzer.managers.Bower$Factory
+com.here.ort.analyzer.managers.Bundler$Factory
```

Comment:
Commit message nit: "the order"
And maybe mention that in this case it does not matter in which order the implementations are loaded.

Reply:
Done.